scripts/pi-hole/js/queries.js

```diff
@@ -115,6 +115,11 @@ $(function () {
         isCNAME = false,
         regexLink = false;
 
+      // accompanies Store domainlist IDs for blocked/permitted queries FTL PR 1409
+      if (data.length > 9 && Number.isInteger(parseInt(data[9], 10)) && data[9] > 0) {
+        regexLink = true;
```

Comment:
```suggestion
        DomainlistLink = true;
```

Reply:
Suggestions:
`listLink` or `itemLink`

Reply:
The suggestion here, was to replace ALL instances of `regexLink` with a different name.

You can choose any name. 
We suggested to use `DomainlistLink`, or `listLink`, or `itemLink` (or whatever you like).

Reply:
```
The suggestion here, was to replace ALL instances of regexLink with a different name.
```
I already did that, and I can see the changes in "Files Changed", 'DomainlistLink' is there 3 times. I've also replaced the modified file on my pihole, everything appears to be working.

![image](https://user-images.githubusercontent.com/24669386/189581548-9639dba4-1a6a-43e5-8a13-e6b727b4124c.png)

![image](https://user-images.githubusercontent.com/24669386/189581687-6c037a7c-dd1b-43a4-8e68-c3730fa7acf3.png)